app/jobs/fs/submit_message_draft_job.rb

```diff
@@ -0,0 +1,24 @@
+class Fs::SubmitMessageDraftJob < ApplicationJob
+  def perform(message_draft, bulk_submit: false, fs_client: FsEnvironment.fs_client)
+    raise "Invalid message!" unless message_draft.valid?(:validate_data)
+
+    fs_api = fs_client.api(box: message_draft.thread.box)
+
+    response = fs_api.post_submission(
+      message_draft.form.identifier,
+      Base64.strict_encode64(message_draft.form_object.content),
+      message_draft.form_object.is_signed,
+      message_draft.form_object.mimetype
+    )
+
+    message_draft.thread.box.message_submission_requests.create(
+      request_url: fs_api.submission_url,
+      response_status: response[:status],
+      bulk: bulk_submit
+    )
+
+    raise RuntimeError.new("Response status is not 202. Message #{response[:body][:errors]}") unless response[:status] == 202
+
+    Fs::SubmitMessageDraftStatusJob.perform_later(message_draft, response[:headers][:location])
```

Comment:
Zvlastny nazov jobu, nie je mi jasne ze co to robi. co to znamena?

Reply:
Isla som podla konvencie ktoru nastavil Marek [tu](https://github.com/slovensko-digital/govbox-pro/pull/394). 
Mame 3 joby:
- vyvolanie asynchronneho submitu
- kontrola stavu submitu (to je tento status)
- spracovanie vysledku submitu,

rovnaka situacia je aj pri asynchronnej validacii.